canonical/juju-dashboard
# WD-11351 - feat: use tag logo

## Done

- Switch to the tag logo

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that the logo on the login screen and side nav are the new tag logo.

## Details

Fixes: #1655.
https://warthogs.atlassian.net/browse/WD-11351

## Screenshots

<img width="374" alt="Screenshot 2024-05-20 at 4 03 10 PM" src="https://github.com/canonical/juju-dashboard/assets/361637/7cdc3cbd-1c0c-4c41-851b-bca88e6c093d">
<img width="407" alt="Screenshot 2024-05-20 at 4 03 23 PM" src="https://github.com/canonical/juju-dashboard/assets/361637/07dcf84f-48de-4c84-a872-b5eb66abba2d">
